### ISSUE TITLE
String tag name as `view`

### ISSUE THREAD

**Issue by Kelin2025** [effector/reflect#60]
Suppose we use pure elements instead of components for forms. We have to do
```ts
const SearchInput = reflect({
  view: (props: InputHTMLAttributes<HTMLInputElement>) => <input {...props} />,
  bind: { /* ... */ }
})
```
Could be just:
```ts
const SearchInput = reflect({
  view: "input",
  bind: { /* ... */ }
})
```

**Comment by Kelin2025** [comment]
@AlexandrHoroshih answering your question - in order to do less view library-specific stuff _(like refs forwarding, types etc.)_, Tailwind users sometimes don't create components but instead just compose classes and apply them to regular HTML elements
So it can be like this:
```tsx
const NameField = reflect({
  view: 'input',
  bind: {
    className: inputEl
  }
})
```tsx
And combined with some utilities
```tsx
const inputEl = cva("px-3 py-2", {
  variants: {
    size: {
      base: "text-base",
      large: "text-3xl"
    }
  }
})

const NameField = reflect({
  view: 'input',
  bind: {
    className: inputEl({ size: $inputSize })
  }
})
```
This way you have fully declarative, isomorphic code, which can be ported to any view library just by replacing `cva` and `reflect` imports

**Comment by AlexandrHoroshih** [comment]
Released in 9.0.0 as `fromTag` helper
